### DISCUSSION TITLE
Changing checkbox field's option value unchecks that box in entries

### DISCUSSION THREAD

**Discussion by modelesque** [craftcms/cms#8737] [Ideas]
### Description
I noticed that by changing the value of a checkbox field's option, any entries that had that option checked are now unchecked, meaning I have to go back through all my entries and recheck that box.

### Steps to reproduce

<img width="1188" alt="Screen Shot 2019-09-19 at 9 23 59 PM" src="https://user-images.githubusercontent.com/13842870/65274984-e5125400-db24-11e9-8887-663c820fc2f8.png">

1. Create a custom checkbox field like the image above.
2. Add the field to an entry, save the entry with "Dancing" checked.
3. Edit the checkbox field by changing the value of the "Dancing" option from `dance` to `dancing`, save changes
4. Go back to the entry and "Dancing" is no longer checked

### Additional info

- Craft version: 3.3.4.1
- PHP version: 7.3
- Database driver & version: mysql

**Comment by brandonkelly** [comment]
Yeah this is because your entry content just stores the actual field option value.  It’s not currently possible for field types to go update the values of any elements that use it, but we should be able to pull that off in Craft 4. So I’ll add this to the v4 milestone.
